content/learn/links.toml

```diff
@@ -19,6 +19,13 @@ image = "/assets/rust-logo-light.svg"
 image_alt = "Rust logo"
 description = "Learn how to use Bevy's types, traits and methods using the in-depth reference documentation, complete with inline examples."
 
+[[links]]
+title = "Contributing Guide"
+url = "/learn/contribute/introduction"
+image = "/assets/book.svg"
+image_alt = "A slanted book"
+description = "Learn how to best contribute to the Bevy Project."
```

Comment:
Right now this is 4th in the list on the "Learn" page. Should it be moved down, or is it fine where it is?

![image](https://github.com/user-attachments/assets/aae15d7e-afd1-4b0d-b2f5-11f00d3a513a)

Reply:
I'm unsure myself, I figured either Alice, or Cart, or really anyone, would chime in with a preference, or if no one had a preference then we'd let it be how it is currently.